en/setup/dev_env_mac.md

```diff
@@ -4,15 +4,10 @@
 
 The installation of Homebrew is quick and easy: [installation instructions](https://brew.sh).
```

Comment:
@LorenzMeier 
1. What does this installation give us the ability to build out of the box - Gazebo sim?, jMAVSim sim?, Nuttx?, Snapdragon?, RaPi?, Bebop?
2. If not one of those, are they "not supported" or do we just have to add instructions?